requirements_test.txt

```diff
@@ -5,7 +5,6 @@ coveralls~=3.3
 coverage~=6.3.3
 pre-commit~=2.19
 pytest-cov~=3.0
-contributors-txt>=0.7.3
```

Comment:
@Pierre-Sassoulas did you want to remove this in another MR?

Reply:
It happened to prevent installing with pip locally I guess it's not that important. (Duplicated entry)